Makefile

```diff
@@ -11,9 +11,13 @@ test:
 release:
 	./gradlew :library:assembleRelease
 
-publish:
-	export IS_LOCAL_DEVELOPMENT=false; ./gradlew :library:uploadArchives
+publish-stable:
+	export IS_LOCAL_DEVELOPMENT=false; ./gradlew :library:bintrayUpload
 
 publish-local:
```

Comment:
This does not work anymore and can be removed.

Reply:
Removed